x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -910,9 +922,12 @@ function renderHelpText(isOverridden: boolean) {
   }
 }
 
-function determineIfDarkModeOverride(settingsClient: IUiSettingsClient) {
-  const isThemeOverridden = settingsClient.isOverridden('theme:darkMode');
-  const isOverriddenThemeDarkMode = settingsClient.get<boolean>('theme:darkMode');
-
-  return isThemeOverridden && isOverriddenThemeDarkMode;
+function determineIfDarkModeOverride(settingsClient: IUiSettingsClient): {
```

Comment:
optional nit: the logic change here makes the function name less correct. Something like `determineIfThemeOverridden` might be a better fit.